include/rekey/rnp_key_store.h

```diff
@@ -155,6 +162,27 @@ pgp_key_t *rnp_key_store_import_key(rnp_key_store_t *,
                                     bool,
                                     pgp_key_import_status_t *);
 
+/**
+ * @brief Get signer's key from key store.
+ *
+ * @param store populated key store, cannot be NULL.
+ * @param sig signature, cannot be NULL.
+ * @return pointer to pgp_key_t structure if key was found or NULL otherwise.
+ */
+pgp_key_t *rnp_key_store_get_signer_key(rnp_key_store_t *store, const pgp_signature_t *sig);
```

Comment:
@rrrooommmaaa Yeah, this is the same, just local variable naming (some other functions use 'store'). Same thing may happen to secret/private key naming, whatever else. Probably it would be refactored to the same thing somewhere in future.